### DISCUSSION TITLE
Cursors

### DISCUSSION THREAD

**Discussion by avrtje** [fhdm-dev/scoppy#23] [General]
Hi fhdm-dev,

In the first place, it is a nice app. Thanks for developing it.
But it should be nice, more than nice, to have a pair of cursors working.
One pair for the amplitude range and one pair for the time range.

Hope that you can make that working.

Best regards,
Jacques

**Comment by fhdm-dev** [comment]
Hi Jacques,

I agree that cursors would be a nice addition to the app. I won't be able to fit them into the next update. Maybe the one after that.
